src/platform/core/src/device/shader/lcd_grid.glsl.hpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2024 fleroviux

```

Comment:
Who is the author of the shader? What is the licence of the shader? Can you please provide a link to the source from which you've ported the shader, so I could suggest how to properly write the header? Authorship and licensing shouldn't be arbitrarily changed, this can be illegal depending on the wording of the original licence. See, e.g., how xBRZ and sharp bilinear shader headers look like.

Reply:
https://github.com/libretro/glsl-shaders/blob/master/handheld/shaders/lcd1x.glsl
https://github.com/jdgleaver/drastic_ds_shaders/blob/master/lcd1x.dsd
I based the port in these sources.  
I already had a header similar to drastic, but it seems I forgot to commit.

Reply:
Yeah, copying either one (and, at your option, adding a line that you've ported it) should suffice.

(I don't understand why it was necessary to apply GPLv2+ to the public domain code, but oh well.)